ios-unit-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: iOS Unit Tests
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+jobs:
+  build:
+    name: Build and Test default scheme using any available iPhone simulator
```

Comment:
Is it possible to run unit tests headless?

Reply:
Good question...from what I can find, you would have to use a third party tool, or some workarounds.  But AFAIK launching the simulator isn't a big time suck...

Reply:
We can just look into it later if build times become an issue